src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/NumFieldElementDriver.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Globalization;
+using Orchard.DynamicForms.Elements;
+using Orchard.Forms.Services;
+using Orchard.Layouts.Framework.Display;
+using Orchard.Layouts.Framework.Drivers;
+using Orchard.Layouts.Helpers;
+using Orchard.Layouts.Services;
+using Orchard.Tokens;
+using DescribeContext = Orchard.Forms.Services.DescribeContext;
+
+namespace Orchard.DynamicForms.Drivers {
+    public class NumFieldElementDriver : FormsElementDriver<NumField>{
```

Comment:
Not very fond of the abbreviation. Maybe use *NumericField* everywhere instead of *NumField*?